components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint/pom.xml

```diff
@@ -31,6 +32,17 @@
     <url>http://wso2.org</url>
 
     <dependencies>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-io</artifactId>
+            <version>1.3.2</version>
```

Comment:
Remove dependency versions from <dependencies>  and add to the dependencyManagement section in the root pom.